server/src/test/java/org/elasticsearch/common/lucene/LuceneTests.java

```diff
@@ -74,7 +77,6 @@
 import static org.hamcrest.Matchers.equalTo;
 
 public class LuceneTests extends ESTestCase {
-    private static final NamedWriteableRegistry EMPTY_REGISTRY = new NamedWriteableRegistry(Collections.emptyList());
```

Comment:
Was this just unused and you're cleaning it up?  It's unclear to me what this removal has to do with the main change.  No worries if you're just cleaning stuff up, just want to make sure I'm not missing something.

Reply:
This was used for serialization / deserialization using the generic method which required a registry.  We cannot use that method anymore because it requires to implement Writable now.